srcpkgs/nemo/template

```diff
@@ -14,7 +14,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.0-or-later"
 homepage="https://github.com/linuxmint/nemo"
```

Comment:
```suggestion
homepage="https://github.com/linuxmint/nemo"
changelog="https://raw.githubusercontent.com/linuxmint/nemo/master/debian/changelog"
```

Reply:
My mistake, I forgot to add the changelog.
Thank you!